optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -274,9 +247,15 @@ def __call__(self, trial):
         val_score = self._get_booster_best_score(booster)
         elapsed_secs = t.elapsed_secs()
         average_iteration_time = elapsed_secs / booster.current_iteration()
+
+        if self.model_dir is not None:
+            path = os.path.join(self.model_dir, "{}.pkl".format(trial.number))
+            with open(path, "wb") as fout:
+                pickle.dump(booster, fout)
```

Comment:
How about logging this operation, including the file path?

Reply:
Thank you for your suggestion. I added it as follows:
> The filenames of the boosters will be ``{model_dir}/{trial_number}.pkl`` (e.g., ``./boosters/0.pkl``).